src/api/user/info/changeSeniorAccount.ts

```
@@ -21,19 +21,17 @@ export const changeSeniorAccont = async ({
+      '/senior/me/account',
+      {
+        phoneNumber,
+        profile,
+        bank,
+        accountHolder,
+        accountNumber,
+        nickName,
+      },
+    );
   } catch (e) {
     throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)